examples/jms-broker/README.md

```diff
@@ -0,0 +1,137 @@
+# WildFly embedded broker used as a remote broker
+
+In this example we are deploying 3 WildFly applications on OpenShift:
+
+* A broker configured with a persistent storage to store not delivered messages.
+* A jms message provider. A servlet that sends messages to a queue.
+* A jms consumer. An MDB EJB that consumes messages.
+
+The builder and runtime images are running JDK11.
```

Comment:
Well, currently we don't scale because that would require a cluster.
Second, the issue is that Artemis can't scale down, we would need to disable all destinations on a pod going down and copy all the messages to the other pods. This is not done at the broker level.